tests/testdata/ca/intermediate/certs/ca-chain.cert.pem

```diff
@@ -0,0 +1,70 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
It is possible to generate these in the tests rather than statically generating the files.

This is fine, but the certs will expire at some point.

Reply:
Agree. We need to fix this for the edge as well.